whipper/common/mbngs.py

```diff
@@ -274,6 +274,19 @@ def _getMetadata(release, discid, country=None):
 # see http://bugs.musicbrainz.org/browser/python-musicbrainz2/trunk/examples/
 #     ripper.py
 
+def getReleaseMetadata(release_id):
+    import musicbrainzngs
+
+    musicbrainzngs.set_useragent("whipper", whipper.__version__,
+                                 "https://github.com/whipper-team/whipper")
+    release = musicbrainzngs.get_release_by_id(
+            release_id, includes=["artists", "artist-credits",
+                                  "recordings", "discids",
+                                  "labels", "recording-level-rels",
+                                  "work-rels", "release-groups"])
+    releaseDetail = release['release']
+    return _getMetadata(releaseDetail)
```

Comment:
This whole function seems to essentially be a simplified `musicbrainz()` function. Wouldn’t it be better to allow `musicbrainz()` to accept `discid = None` and pass that through? Or did you already consider or try this?

Reply:
Thanks for your review @Freso. 

Yes i did think about allowing `musicbrainz()` to accept `discid=None` but i just thought (maybe it's not right) that that might make `muscibrainz()` has too many functionality and therefore makes whipper less scalable for example whipper might want to lookup for other information as well in the future and there will be tons of `if...else...` (just what i thought).